### ISSUE TITLE
apiVersion PodDisruptionBudget is incorrect

### ISSUE THREAD

**Issue by ismailyenigul** [kontena/lens#6503]
**Describe the bug**

Deployed PodDisruptionBudget as apiVersion: policy/v1  but when I try to edit this resource, Lens IDE shows it as  `apiVersion: policy/v1beta1`
What I see with kubectl
```
kubectl get poddisruptionbudgets.policy -n cluster-autoscaler eks-ca-aws-cluster-autoscaler -o yaml     
apiVersion: policy/v1
kind: PodDisruptionBudget
metadata:
  annotations:
    meta.helm.sh/release-name: eks-ca
    meta.helm.sh/release-namespace: cluster-autoscaler
  creationTimestamp: "2022-11-02T11:43:19Z"

```

**To Reproduce**

Deploy a PodDisruptionBudget with apiVersion: v1
Edit it on Lens IDE

**Expected behavior**
It should display as v1 not v1beta

**Screenshots**
<img width="939" alt="image" src="https://user-images.githubusercontent.com/2566232/199500068-d944b80a-4488-4570-8888-5f8e2fc1d234.png">

**Environment (please complete the following information):**
- Lens Version:  Lens: 2022.10.311317-latest

- OS: MacOS

**Comment by Nokel81** [comment]
This is a consequence that kubernetes doesn't store the version that a resource was created with internally and will convert something on the fly to a version that is requested.

#4406 might help mitigate this slightly

**Comment by mloskot** [comment]
I'm using Lens 2023.1.110749-latest on Windows 11.
I'm connecting to AKS cluster based on Kubernetes 1.25.5 which I recently migrated minor-by-minor from 1.22.
As per https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

> The policy/v1beta1 API version of PodDisruptionBudget is no longer served as of v1.25

So, I also updated my PDB-s to use `policy/v1`

Lens stopped listing the PDB-s and is warning instead, which, I guess, may be related to this issue:

![image](https://user-images.githubusercontent.com/80741/218724813-77e3f6cf-d983-4357-ad59-c6f0cda9f7e9.png)

**Comment by Nokel81** [comment]
This was fixed by #6989 and will be released in the next release

**Comment by mloskot** [comment]
@Nokel81  Thank you!
